README.md

```diff
@@ -4,41 +4,6 @@
 [![Coverage Status](https://coveralls.io/repos/github/lkilcher/dolfyn/badge.svg?branch=master)](https://coveralls.io/github/lkilcher/dolfyn?branch=master)
 [![Documentation Status](https://readthedocs.org/projects/dolfyn/badge/?version=latest)](https://dolfyn.readthedocs.io/en/latest/?badge=latest)
 
-BIG NEWS!!!
```

Comment:
I think we should retain some of this text somewhere, perhaps below the summary section, and with fewer "!".